docs/program/apis/data-client.md

```diff
@@ -74,6 +74,66 @@ The data client API supports the following methods (among [others](https://pytho
 
 {{< readfile "/static/include/services/apis/data-client.md" >}}
 
+### TabularDataByFilter
+
+Retrieve optionally filtered tabular data from the [Viam app](https://app.viam.com).
+Tabular data can be found under the **Sensors** subtab of the app's [**Data** tab](https://app.viam.com/data).
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `filter` [(Optional[viam.proto.app.data.Filter])](https://python.viam.dev/autoapi/viam/proto/app/data/index.html#viam.proto.app.data.Filter): Optional `Filter` specifying tabular data to retrieve. Specify no filter to download all tabular data.
+- `dest` [(Optional[str])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Filepath to write retrieved data to.
+
+**Returns**:
+
+- [(List[TabularData])](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient.TabularData): The tabular data retrieved from the [Viam app](https://app.viam.com).
+
+```python {class="line-numbers linkable-line-numbers"}
+# from viam.proto.app.data import Filter
+
+my_filter = Filter(component_name="left_motor")
+tabular_data = await data_client.tabular_data_by_filter(my_filter)
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient.tabular_data_by_filter).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### BinaryDataByFilter
+
+Retrieve optionally filtered binary data from the [Viam app](https://app.viam.com).
+Binary data can be found under the **Images**, **Point clouds**, or **Files** subtab of the app's [**Data** tab](https://app.viam.com/data), depending on the type of data that you have uploaded.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `filter` [(Optional[viam.proto.app.data.Filter])](https://python.viam.dev/autoapi/viam/proto/app/data/index.html#viam.proto.app.data.Filter): Optional `Filter` specifying binary data to retrieve. Specify no filter to download all binary data.
+- `dest` [(Optional[str])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Filepath to write retrieved data to.
+- `include_file_data` [(bool)](https://docs.python.org/3/c-api/bool.html#boolean-objects): Boolean specifying whether to include the binary file data with each retrieved file. Defaults to `true`, where both the files’ data and metadata are returned.
+- `num_files` [(Optional[str])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Number of binary data to return. Passing `0` returns all binary data matching the filter. Defaults to `100` if no binary data is requested, otherwise `10`.
+
+**Returns**:
+
+- [(List[BinaryData])](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient.BinaryData): The binary data retrieved from the [Viam app](https://app.viam.com).
+
+```python {class="line-numbers linkable-line-numbers"}
+# from viam.proto.app.data import Filter
```

Comment:
Hmm so this is on purpose. Hmm I kind of like it uncommented because then if they copy it into their code it would give a lint error to put it at the top probably?

Reply:
Good with me, that makes sense! Uncommenting